### ISSUE TITLE
prefix arg does not work

### ISSUE THREAD

**Issue by delihus** [UniversalRobots/Universal_Robots_ROS2_Ignition_Simulation#19]
On branch humble
```bash
ros2 launch ur_simulation_gz ur_sim_control.launch.py prefix:=ur3
```

```
[robot_state_publisher-1] [INFO] [1714049311.199665793] [robot_state_publisher]: got segment base
[robot_state_publisher-1] [INFO] [1714049311.199778913] [robot_state_publisher]: got segment base_link
[robot_state_publisher-1] [INFO] [1714049311.199791699] [robot_state_publisher]: got segment base_link_inertia
[robot_state_publisher-1] [INFO] [1714049311.199806512] [robot_state_publisher]: got segment flange
[robot_state_publisher-1] [INFO] [1714049311.199816643] [robot_state_publisher]: got segment forearm_link
[robot_state_publisher-1] [INFO] [1714049311.199826006] [robot_state_publisher]: got segment ft_frame
[robot_state_publisher-1] [INFO] [1714049311.199836067] [robot_state_publisher]: got segment shoulder_link
[robot_state_publisher-1] [INFO] [1714049311.199845709] [robot_state_publisher]: got segment tool0
[robot_state_publisher-1] [INFO] [1714049311.199855561] [robot_state_publisher]: got segment upper_arm_link
[robot_state_publisher-1] [INFO] [1714049311.199865622] [robot_state_publisher]: got segment world
[robot_state_publisher-1] [INFO] [1714049311.199875334] [robot_state_publisher]: got segment wrist_1_link
[robot_state_publisher-1] [INFO] [1714049311.199885256] [robot_state_publisher]: got segment wrist_2_link
[robot_state_publisher-1] [INFO] [1714049311.199895107] [robot_state_publisher]: got segment wrist_3_link
```

**Comment by fmauch** [comment]
Thanks for reporting this. We migrated towards using `tf_prefix` everywhere but apparently things got lost on the way.

**Comment by urfeex** [comment]
I think, this got fixed in #13
